build.sbt

```diff
@@ -11,7 +11,9 @@ resolvers ++= {
   else Seq.empty
 }
 
-libraryDependencies += "org.scoverage" %% "scalac-scoverage-plugin" % "1.2.0-SNAPSHOT"
+libraryDependencies += "org.scoverage" %% "scalac-scoverage-plugin" % "1.2.0"
+
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "0.6.12")
```

Comment:
@lustefaniak Basically, I fully agree. In fact, I answered this comment on the [SBT gitter](https://gitter.im/sbt/sbt?at=57d81ae2b37816b14fb37c66) even before you posted this comment - and even [referenced your work](https://gitter.im/sbt/sbt?at=57d86772b8d4c138408e9de8)! So yes, we are on the same page.

Note the other suggestion of having a plugin per platform ( one just extending the other) as we do with libs.

The reason I have it as it is now is "just to get it working" - note that I am also enforcing a minimum version of scala.js of `0.6.12` as that is the version that has scala 2.12.0-RC1. This will also be the one that scoverage/scalac-scoverage-plugin#177 uses. Which is the other point - what if the version of scala.js an end user uses is diff to the one that scalac-scoverage-plugin uses? Or .... 

These are all points to be considered, really this PR just aims to be a minimum "that works"